modules/bitcoind.nix

```diff
@@ -117,6 +119,27 @@ in {
           '';
         };
       };
+      rpcbind = mkOption {
+        type = types.lines;
```

Comment:
Would be more consistent and simpler if this was just a list of ip addresses (same with `rpcallowip` and in the liquid module)

Reply:
See `fixup! liquid: make rpcbind and rpcallowip lists of ip addresses` and `fixup! bitcoind: make rpcbind and rpcallowip lists of ip addresses`